README.md

```diff
@@ -386,6 +386,71 @@ $ npx electron-build-env nj-cli build --release
 
 otherwise you will get dreaded  `A dynamic link library (DLL) initialization routine failed` when importing the rust module in electron
 
+## Preparing npm packages
+
+Node module generated with `node-bindgen` can be used directly in any node JS project, just copied `index.node` into it. But in case of direct access to a module IDE will not highlight available functions, classes etc. Usually, this is not comfortable and makes the risks of potential bugs higher as soon as the public API of the node module is changed.
+
+To create a full-fledged npm package with TypeScript types definitions and all necessary JavaScript wrappers can be used a crate `tslink`.
```

Comment:
maybe we should add a link to the tslink repo here?

Reply:
Added link to crates.io, which has links to documentation and to github repo